dv/cosim/spike_cosim.cc

```diff
@@ -440,6 +438,21 @@ void SpikeCosim::leave_nmi_mode() {
 #endif
 }
 
+void SpikeCosim::initial_proc_setup(uint32_t start_pc, uint32_t start_mtvec) {
+  processor->get_state()->pc = start_pc;
+  processor->get_state()->mtvec->write(start_mtvec);
+
+  processor->get_state()->csrmap[CSR_MARCHID] =
+      std::make_shared<const_csr_t>(processor.get(), CSR_MARCHID, IBEX_MARCHID);
+
+  processor->set_mmu_capability(IMPL_MMU_SBARE);
+
+  for (int i = 0; i < processor->TM.count(); ++i) {
+    processor->TM.tdata2_write(processor.get(), i, 0);
+    processor->TM.tdata1_write(processor.get(), i, 0x28001048);
+  }
```

Comment:
I'm assuming these are debug triggers, are these values documented in Ibex somewhere?

Reply:
They are debug triggers. The values are documented as the reset values here: https://ibex-core.readthedocs.io/en/latest/03_reference/cs_registers.html#trigger-data-register-1-tdata1 though now I look at that TDATA1 should maybe have a different value written to it at start for cosim (or our documentation needs a fix). I'll merge this but take a look and do an appropriate follow up PR.